contracts/staking/contracts/src/staking_pools/MixinCumulativeRewards.sol

```diff
@@ -51,40 +57,52 @@ contract MixinCumulativeRewards is
         pure
         returns (bool)
     {
-        // we use the denominator as a proxy for whether the cumulative
+        // We use the denominator as a proxy for whether the cumulative
         // reward is set, as setting the cumulative reward always sets this
         // field to at least 1.
         return cumulativeReward.denominator != 0;
     }
 
-    /// Returns true iff the cumulative reward for `poolId` at `epoch` can be unset.
+    /// @dev Returns true iff the cumulative reward for `poolId` at `epoch` can
+    ///      be unset.
     /// @param poolId Unique id of pool.
-    /// @param epoch of the cumulative reward.
+    /// @param epoch Epoch of the cumulative reward.
     function _canUnsetCumulativeReward(bytes32 poolId, uint256 epoch)
         internal
         view
         returns (bool)
     {
-        return (
-            _isCumulativeRewardSet(_cumulativeRewardsByPool[poolId][epoch]) &&   // is there a value to unset
-            _cumulativeRewardsByPoolReferenceCounter[poolId][epoch] == 0 &&      // no references to this CR
-            _cumulativeRewardsByPoolLastStored[poolId] > epoch                   // this is *not* the most recent CR
-        );
+        // Must be a value to unset
+        if (!_isCumulativeRewardSet(_cumulativeRewardsByPool[poolId][epoch])) {
+            return false;
+        }
+        // Must be no references to this CR
+        if (_cumulativeRewardsByPoolReferenceCounter[poolId][epoch] != 0) {
+            return false;
+        }
+        // Must not be the most recent CR.
+        if (_cumulativeRewardsByPoolLastStored[poolId] == epoch) {
```

Comment:
Should this be `<=`?

Reply:
No, it's normal operation to clear CRs from past epochs. We just special case the current epoch because the system breaks if we don't have it.

Reply:
Comment should be updated.